tests/test_utils.py

```diff
@@ -0,0 +1,19 @@
+from pidtree_bcc import utils as u
```

Comment:
put `from`s after `import`s. Ditto about just leaving the name as `utils`

Reply:
Fixed in 90143be5f4d3710dd3dab6e2f3e800c3f2cc9189